src/OpenTelemetry.AutoInstrumentation/Instrumentations/MySqlData/MySqlConnectionStringBuilderIntegration.cs

```diff
@@ -33,7 +33,9 @@ namespace OpenTelemetry.AutoInstrumentation.Instrumentations.MySqlData;
     IntegrationName = "MySqlData")]
 public class MySqlConnectionStringBuilderIntegration
 {
+#if !NETFRAMEWORK
```

Comment:
```suggestion
#if NETCOREAPP3_1_OR_GREATER
```
Easier to understand I think.

Reply:
I will check all instrumentations later.

Reply:
I prefer `!NETFRAMEWORK` as it requires less maintenance e.g. when we will drop `.NET Core 3.1`. Also it is shorter to type and  easier to remember 😉